hs_modelinstance/views.py.old

```diff
@@ -4,7 +4,7 @@ from django.views import generic
 from django.http import HttpResponse, HttpResponseRedirect
 from django.core.urlresolvers import reverse
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render_to_response
+#from django.shortcuts import render_to_response
```

Comment:
@sblack-usu just remove this import rather than comment it out?

Reply:
do you know what this file is for?  I really should be able to just remove the whole file (.py.old)

Reply:
I don't know what is file is for, but I am very sure this whole file should be removed, so just go ahead to remove the whole file.

Reply:
removed